pkgs/tools/admin/azure-cli/extensions-generated.nix

```diff
@@ -839,6 +839,13 @@
     sha256 = "ab19142c9a2e06190b6dac272d8cf29e179c9e283f965f8e3a4d29275b847ce5";
     description = "Microsoft Azure Command-Line Tools SiteRecovery Extension";
   };
+  ssh = mkAzExtension rec {
+    pname = "ssh";
+    version = "1.1.3";
```

Comment:
Thanks for this PR! How did you choose `2.26.0` as the version to use in the `query-extension-index.sh` script? I think 2.60.0 is the latest CLI version included in nixpkgs. This is probably why `1.1.3` is the version for the ssh extension is chosen, and not 2.0.5: https://github.com/Azure/azure-cli-extensions/blob/main/src/ssh/setup.py#L10

Reply:
Hey good question! I think it is a typo. I checked the release notes page and the latest version is 2.63.0

https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli

I'll go ahead and re-run the script with that version, thanks for the catch!